hack/run-tests.sh

```diff
@@ -3,36 +3,73 @@
 set -e
 
 TEST_TYPE=$1
-if [ "$TEST_TYPE" != "integration" ] && [ "$TEST_TYPE" != "e2e" ]; then
-    echo "Invalid test type: $TEST_TYPE. Please provide either 'integration' or 'e2e' as the first argument."
+if [ "$TEST_TYPE" != "integration" ] && [ "$TEST_TYPE" != "e2e" ] && [ "$TEST_TYPE" != "performance" ]; then
+    echo "Invalid test type: $TEST_TYPE. Please provide either 'integration', 'e2e' or 'performance' as the first argument."
     exit 1
 fi
 
-# Cleanup the kind cluster when the script exits
+YHS_SERVER=$(YHS_SERVER:-http://localhost:8989}
```

Comment:
```suggestion
YHS_SERVER=${YHS_SERVER:-http://localhost:8989}
```

this typo breaks the script

Reply:
fixed